doc/plugins/org.polarsys.capella.ui.doc/html/Installation Guide/How to install Capella and Addons.mediawiki

```diff
@@ -1,236 +1,272 @@
+* Users can now use the setup properly. If afterwards, you want to install new addons to all contexts, just run Capella and install them, for one particular context, run Capella with one of the <code>contextX_admin.ini</code>.
+
+=== Recommendations for setting up multi-users configurations ===
+
+'''Ensure that addons are properly installed before users encounter installation errors'''
+
+After updating the installation of Capella or addons
+
+* Do not forget to restart the application as requested while installation of Update sites, or by adding the option <code>-clean</code> in command line parameter.
+
+* It is recommended to launch once all your users client by adding a <code>-clean</code> in command line argument to ensure that configurations are updated properly for each users then check Error Log messages. Do not forget to remove this <code>-clean</code> parameter after this test to avoid starting performance issues.
+
+* It is recommended to start the application once with <code>-checkConfiguration</code> to ensure that the cache is up to date with the respect to the contents of the installed bundles.
+
+* It is recommended to switch to a new workspace, some information might have been cached in it (Java version, etc...).
+
+'''One user can be part of several working contexts'''
+
+When working with several contexts, it is important to avoid mixing configuration between contexts for both <code>-Dosgi.sharedConfiguration.area</code> and <code>-Dosgi.configuration.area</code>. Both path must contain a <code>contextX</code> information. Afterwards, if a user is part of a new <code>contextY</code>, administrator will just have to provide a new launcher with <code>--launcher.ini contextY.ini</code> and the configuration will be located in a dedicated <code>contextY</code> folder and will not be not mixed up with previous contexts.
+
+'''Parent folder of configuration shall be dedicated to capella'''
+
+* Addons will be installed in the parent folder of the configuration area. When setting up <code>-Dosgi.configuration.area</code> or <code>-Dosgi.sharedConfiguration.area</code>, it is important to have a parent folder dedicated to capella. (<code>@user.home/capella-configuration</code> is a bad choice as <code>@user.home</code> will be polluted with other folders, whereas setting up <code>-Dosgi.configuration.area=somewhere/capella/configuration</code> is recommended, as addons will be installed in <code>somewhere/capella/(configuration|p2|plugins|features)</code>.
+
+'''Users can install its own addons'''
+
+It important to keep in mind that with all the described configurations above, an user can install its own addons in its configuration area. This can be useful, or not expected. This can't be prevented by setting up the configuration, but it can be setup by setting read-only access to some predefined folders such as <code>@user.home/xxx/capella/(p2|plugins|features|dropins)</code>.

```

Comment:
It is important...
This can be useful but sometimes not an expected installation (e.g. imposed by administrators)